did/plugins/gitlab.py

```diff
@@ -169,6 +176,17 @@ def get_project_mr(self, project_id, mr_id):
         mrs = self.get_project_mrs(project_id)
         mr = next(filter(lambda x: x['id'] == mr_id, mrs), None)
         return mr
+    
+    def get_user_mr(self, username, state, since, until):
+        since = since.date.strftime('%Y-%m-%dT%H:%M:%S.000Z')
+        until = until.date.strftime('%Y-%m-%dT%H:%M:%S.000Z')
+        endpoint = 'merge_requests'
+        return self._get_gitlab_api_list(endpoint, params={
+            'author_username': username,
+            'state': state,
+            'updated_after': since,
+            'updated_before': until
```

Comment:
Is it intentional not setting `get_all_results=True` here? 
Only the first page of results will be fetched here.